web/themes/custom/recipe_theme/src/sass/block/_block.scss

```diff
@@ -0,0 +1,12 @@
+.layout__region {
+  h2 {
```

Comment:
These rules apply to all the `h2` headings in `layout__region`. Is this intentional?

Reply:
> These rules apply to all the `h2` headings in `layout__region`. Is this intentional?